packages/main/src/MenuItem.ts

```diff
@@ -158,6 +158,15 @@ class MenuItem extends ListItem {
 	@slot({ "default": true, type: HTMLElement, invalidateOnChildChange: true })
 	items!: Array<MenuItem>;
 
+	/**
+	 * Defines the components that should be displayed at the end of the menu item.
+	 *
+	 * **Note:** It is highly recommended to slot only components of type `ui5-button`,`ui5-link` or `ui5-icon`.
```

Comment:
please add the standard wording we have for these slots "in order to preserve the intended design" at the end of the sentence.

Reply:
Done.